cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -1,19 +1,32 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
 import pytest
+from widgetastic.utils import partial_match
 
 from cfme.configure.configuration.region_settings import Tag, Category
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.provisioning import do_vm_provisioning
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance import ViaSSUI, ViaUI
 from cfme.utils.generators import random_vm_name
 from cfme.utils.update import update
 
+
 pytestmark = [
-    pytest.mark.provider([RHEVMProvider, VMwareProvider], scope="module")
+    pytest.mark.provider([RHEVMProvider, VMwareProvider], scope="module", selector=ONE_PER_TYPE)
 ]
 
 
+@pytest.fixture
+def admin_email(appliance):
+    """Required for user quota tagging services to work, as it's mandatory for it's functioning."""
+    user = appliance.collections.users
+    admin = user.instantiate(name='Administrator')
+    admin.update({'email': 'xyz@redhat.com'})
```

Comment:
**REQUIRED**
Please context manager to update an entity. `User` entity inherits `Updatable` so it should look like this:
```python
with update(admin):
    admin.email = 'xyz@redhat.com'
```
Please add teardown.